services/app-api/serverless.yml

```diff
@@ -169,21 +166,6 @@ functions:
           cors: true
           authorizer: aws_iam
 
-  postgres:
-    handler: handlers/postgres_check.main
-    events:
-      - http:
-          path: postgres_check
-          method: get
-          cors: true
-    vpc:
-      securityGroupIds:
-        - ${self:custom.sgId}
-      subnetIds: ${self:custom.privateSubnets}
-    layers:
-      - !Ref PrismaClientLambdaLayer
-      - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-0-1:1
-
```

Comment:
we never use this function. I had made it when we were just getting prisma up and running, but by now we have coverage of postgres vis prisma with the graphql func.